src/main/java/org/opentripplanner/hasura_client/mappers/VehiclePositionsMapper.java

```diff
@@ -7,12 +7,16 @@
 import org.slf4j.LoggerFactory;
 
 import java.math.BigDecimal;
+import java.util.HashMap;
+import java.util.Map;
 import java.util.Optional;
 
 public class VehiclePositionsMapper extends HasuraToOTPMapper<Vehicle, VehicleDescription> {
 
     private static final Logger LOG = LoggerFactory.getLogger(HasuraGetter.class);
 
+    private Map<Provider, Long> numberOfMappedVehiclesPerProvider = new HashMap<>();
```

Comment:
Wouldn't Set<Provider> be more straight forward? Mapping from providers to number of fetched vehicles isn't needed anywhere.

Reply:
Good idea - done. My initial plan was to perhaps also take into account, how many of the vehicles have disappeared, however at this point the exact number of vehicles per provider is indeed not required.